### PR TITLE
Settings on right drawer

### LINKED ISSUE #711: Unified settings pane
Rather than having settings spread across modals, dropdowns, tooltips, etc, where nobody can find them, why not something like this? (MS Paint mockups, thanks @ysofi for the idea)

Closed (reduced top-bar clutter - just the universal "settings" gear):
quran-settings-closed
Open:
quran-settings-open

The existing settings modal would be eliminated, along with its entry in the left nav. This does move stuff like reading mode and night mode from 1 click to 2 clicks, but it eliminates confusion over where *all* the settings are, and re-surfaces the translation and recitation options that are very well hidden at the moment. It also adds Yet Another Side-Menu - which I think is the lesser of two evils compared to the current modal settings screen and mystery-meat option buttons.

### PR DESCRIPTION
**Pull request by mmahalwy**
Fixes #711

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32771

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32772

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32776

File: src/components/ContentDropdown/index.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-import ButtonToolbar from 'react-bootstrap/lib/ButtonToolbar';
-import DropdownButton from 'react-bootstrap/lib/DropdownButton';
-import MenuItem from 'react-bootstrap/lib/MenuItem';
 import LocaleFormattedMessage from 'components/LocaleFormattedMessage';
 import { loadTranslations } from 'redux/actions/options';
 import { contentType } from 'types';
+import Menu, { MenuItem } from 'quran-components/lib/Menu';
+import Checkbox from 'quran-components/lib/Checkbox';
+import Loader from 'quran-components/lib/Loader';
+import Icon from 'quran-components/lib/Icon';
```
Comment (sabeurthabti) [inline src/components/ContentDropdown/index.js]: this is a beautiful sight :D

**Comment by sabeurthabti** [comment]
minor issue : screen shot 2017-04-07 at 2 26 14 pm

**Comment by naveed-ahmad** [comment]
Looks good! Few UI issues: 

missing translations:



Translation names aren't alinged

File: src/components/ContentDropdown/index.js (hunk 1 above)
Comment (mmahalwy) [inline src/components/ContentDropdown/index.js]: +1

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32782

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32783

**Comment by ysofi** [comment]
@sabeurthabti 

Do you mean something like below for reset font size?  As in show the default as 100% and decreasing and increasing it should alter the % accordingly. If the user wants to go back to the default, they set it back 100%. This is assuming that 100% is clear as "default".

The alternative, "reset font" or similar, is there an ideal placement?

font_size

**Pull request merged**